chapters/helloworld/index.rst

```diff
@@ -82,7 +96,7 @@ And the implementation behind it? It's simple:
     .. image:: hello-world-flow.png
        :align: center
 
-All the big words you can name: computer vision, neural IR, microservice, message queue, elastic, replicas, and shards all happened in just one minute!
+All the big words you can name: computer vision, :term:`neural search, microservice, message queue, elastic, and :term:`shards` all happened in just one minute!
```

Comment:
What does elastic mean in this context? It only makes me think of Elastic Search. Perhaps we should include in glossary and link accordingly

@hanxiao ?

Reply:
I was also going to ask about this. I wasn't sure what is this about, it was on the previous Hello-world